konker.registry/src/main/java/com/konkerlabs/platform/registry/config/SecurityConfig.java

```diff
@@ -106,6 +106,16 @@ protected void configure(HttpSecurity http) throws Exception {
                                                 .getHeader(item.getValue().replaceAll("[\\{\\}]", "")))
                                                 .map(origin -> origin)
                                                 .orElse("*"));
+                            } else if (item.getKey().equals("Access-Control-Allow-Methods") && 
+                            		httpServletRequest.getRequestURI().contains("/pub")) {
```

Comment:
This "contains" may clash with random generated apiKeys. If an apiKey starts with "pub" and we are subscribing, it mail fail